intro_to_first_principles/README.md

```diff
@@ -0,0 +1,381 @@
+| **27 Processing (P)**   | **33** `compute` (P) `secrecy` (C)             | **35** `CPU calculations` (P) `incorrect values` (I) | **37** `slow and freezes` (A) `run application` (P) |
+
+
+*Combo 2: States vs. Countermeasures*
+
+| **Game 2**              | *28* Education (E)                     | *29* Policy (Po)                      | *30* Technology (Te)                              |
+|-------------------------|----------------------------------------|---------------------------------------|---------------------------------------------------|
+| **25 Storage (S)**      | **38** `files` (S) `training` (E)        | **36** `rules` (Po) `backups` (S)       | **31** `hard drive` (S) `AES encryption` (Te)       |
+| **26 Transmission (T)** | **32**  `send email` (T) `unsure` (E)    | **39** `recovery plan` (Po) `email` (T) | **34** `email` (T) `digital signature` (Te)         |
+| **27 Processing (P)**   | **35** `compute` (P) `reads article` (E) | **33** `compute` (P) `rule` (Po)        | **37** `powerful device` (Te) `run application` (P) |
+
+*Combo 3: Characteristics vs. Countermeasures*
+
+| **Game 3**                 | *28* Education (E)                        | *29* Policy (Po)                                              | *30* Technology (Te)                                 |
+|----------------------------|-------------------------------------------|---------------------------------------------------------------|------------------------------------------------------|
+| **22 Confidentiality (C)** | **32** `unsure` (E) `no one can read` (C)   | **33** `secrecy` (C) `rules` (Po)                               | **31** `sensitive data` (C) `AES encryption` (Te)      |
+| **23 Integrity (I)**       | **35**  `incorrect` (I) `reads article` (E) | **36** `not altered` (I) `rules` (Po)                           | **34** `digitally signed` (Te) `digital signature` (I) |
+| **24 Availability (A)**    | **38** `find files` (A) `training` (E)      | **39** `constant email communications` (A) `recovery plan` (Po) | **37** `powerful device` (Te) `slow and freezes` (A)   |
+
+If there is a question as to where the card should be placed the answer is on the card – encrypted. For example on card 31 the lower right hand has the code XON21. This code is a simple rotation cipher and it rotated 21. You will get a three letter answer. In this case it will be CTS. The first letter will be either C, I, or A for Confidentiality, Integrity, Availability. The second letter will be P, E, or T for safeguards – Policy, Education or Technology. The third letter will be S, T or P for states – Storage, Transmission or Processing. In this case the answer is C for Confidentiality, and S for Storage.
+
+
+## Additional Resources
+
+- [NSA document listing Cybersecurity first principles](https://users.cs.jmu.edu/tjadenbc/Bootcamp/0-GenCyber-First-Principles.pdf)
```

Comment:
maybe we can download and host the resources in an assets folder in the repo?

Reply:
Yes. That is exactly how I have it setup.